### ISSUE TITLE
Failed to load response

### ISSUE THREAD

**Issue by NoahSamson** [ClimenteA/flaskwebgui#22]
I am getting this error, when I try to run using flaskwebgui. It works fine when using it in chrome.

![Capture](https://user-images.githubusercontent.com/32417063/80920482-d44de000-8d8d-11ea-8b11-584438bfca9f.JPG)

I set the header to not use cache, other than that the code is similar

![image](https://user-images.githubusercontent.com/32417063/80920529-16772180-8d8e-11ea-992e-a2964049e439.png)

![image](https://user-images.githubusercontent.com/32417063/80920519-05c6ab80-8d8e-11ea-9e7d-cca39338b030.png)

![image](https://user-images.githubusercontent.com/32417063/80920536-2b53b500-8d8e-11ea-88d5-5a8f6d5737cd.png)

When running this, it opens in the normal browser with the link, but there is no pop up with gui.
![image](https://user-images.githubusercontent.com/32417063/80920569-5a6a2680-8d8e-11ea-8c29-bf39e06519d9.png)

Any help would be appreciated

**Comment by ClimenteA** [comment]
Try these bellow:

The `setInterval` function isn't able to make a request to flask local server, maybe port 5001 is in use, try to change it and see if it works.

I see you set the host to serve all incoming requests for that in the js should be the ip of your pc instead of localhost.

Remove `ui = FlaskUI(app)` and add it bellow like:
```
if __name__ == "__main__":
    FlaskUI(app).run()
```

**Comment by NoahSamson** [comment]
> Try these bellow:
> 
> The `setInterval` function isn't able to make a request to flask local server, maybe port 5001 is in use, try to change it and see if it works.
> 
> I see you set the host to serve all incoming requests for that in the js should be the ip of your pc instead of localhost.
> 
> Remove `ui = FlaskUI(app)` and add it bellow like:
> 
> ```
> if __name__ == "__main__":
>     FlaskUI(app).run()
> ```

I tried using changing the port 5001 to 5002 and 5003 to check, but I still got the same error. I checked if the port is being used, and only the port 5000 is being used and that leads to the localhost page in browser, which is working fine and does not open the gui.
![image](https://user-images.githubusercontent.com/32417063/80991866-4f7ac900-8e56-11ea-93b0-dd31723d4456.png)

I changed the localhost in js to the ip of the pc, but still got the same error.

**Comment by ClimenteA** [comment]
What remained is the js code that is not allowed to run `ERR_CONNECTION_REFUSSED`, something is blocking the request from the js.

The example from this flaskwebgui repo works?

**Comment by NoahSamson** [comment]
I was able to run it when I started the project, but it stopped working now.

When i try to run the examples here I get this error:
![image](https://user-images.githubusercontent.com/32417063/80995873-94a1f980-8e5c-11ea-8154-b601bf3fcddd.png)

**Comment by NoahSamson** [comment]
> I was able to run it when I started the project, but it stopped working now.
> 
> I just ran the examples in this repo.
> I got the same error.
> ![image](https://user-images.githubusercontent.com/32417063/80996587-adf77580-8e5d-11ea-9db3-97c43023a722.png)
> 
> I am not sure why the request from the js is blocked.

**Comment by ClimenteA** [comment]
Try to clear all cache go to: chrome://settings/clearBrowserData 
If it doesn't work give me the link to the github repo, I will try to reproduce the error on my side.

**Comment by NoahSamson** [comment]
> Try to clear all cache go to: chrome://settings/clearBrowserData
> If it doesn't work give me the link to the github repo, I will try to reproduce the error on my side.

I tried clearing all cache, but still got the same error.
I have uploaded the project to github
https://github.com/NoahSamson/FYPCLASS
I have removed the models in the model folder since their size is too large.

Thank you so much.

**Comment by ClimenteA** [comment]
I commented out the code related to server class and added bellow for test
```
@app.route("/")
def index():
    return "<h1>It works!</h1>"
```
It worked for me, I didn't got the error you got. 
You can download/fork the source code from this repository (flaskwebgui it's just one file checkout `src` folder)  change it for your needs.

If you find the problem please update this issue.

**Comment by NoahSamson** [comment]
> I commented out the code related to server class and added bellow for test
> 
> ```
> @app.route("/")
> def index():
>     return "<h1>It works!</h1>"
> ```
> 
> It worked for me, I didn't got the error you got.
> You can download/fork the source code from this repository (flaskwebgui it's just one file checkout `src` folder) change it for your needs.
> 
> If you find the problem please update this issue.

I also tried to removing the server class code, this time I do not get the ERR_CONNECTION_REFUSSED error in the console, But the gui still does not pop up.
Were you able to get the gui to work with my code when you removed the server class code?

**Comment by ClimenteA** [comment]
Yes, I attached the file here:
[app.txt](https://github.com/ClimenteA/flaskwebgui/files/4593519/app.txt)

**Comment by ClimenteA** [comment]
Released a new version(0.1.15+), hopefully it fixes this issue.
